src/mono/mono/utils/mono-context.h

```diff
@@ -11,6 +11,14 @@
 #ifndef __MONO_MONO_CONTEXT_H__
 #define __MONO_MONO_CONTEXT_H__
 
+/* 
+ * Handle non-gnu libc versions with nothing in features.h 
+ * We have no idea what they're compatible with, so always fail.
+ */
+#ifndef __GLIBC_PREREQ
```

Comment:
Probably better to just change its (single) usage below:

```diff
- #if __GLIBC_PREREQ(2, 26)
+ #if defined(__GLIBC_PREREQ) && __GLIBC_PREREQ(2, 26)
```
or `!defined(__GLIBC_PREREQ) ||` depending on the availability of `struct ucontext`.

Reply:
Pushed